lib/debugger.coffee

```diff
@@ -23,19 +23,28 @@ class ProcessManager extends EventEmitter
     result[key(e)] = value(e) for e in env.split(";")
     return result
 
-  start: (file) ->
+  startActiveFile: () ->
+    @start true
+
+  start: (withActiveFile) ->
+    startActive = withActiveFile
     @cleanup()
       .then =>
+        packageJSON = require @atom.project.resolvePath('package.json')
```

Comment:
that's interesting.  I get the usefulness of that.  But i'm not sure the debugger should make that assumption.

You're right.  Assuming the project is structured that way- it makes sense to try and debug that specific package/app.
But i think coupling the functionality of the debugger to that specific file structure maybe a bit too prescriptive.

What if the `atom-project/` folder above- had it's own `package.json` in the root?  Would you stop at the first `package.json`, as you walk the directories upward?
What if the open file in the editor is a completely unrelated file?  Like the notes file on my desktop.

To take a step back a bit-  I think it's an important feature when it comes to having a multi project debugging environment.  And something i've actually been thinking about taking a stab at.

I also think the first step in that direction- might need to be a way to support multiple debug configurations in a single atom session.  Or the assumption would be that all the projects in the editor, would be configured/structured exactly the same way.
I'm not sure that's a great stipulation for debugging multiple projects in the same editor session.

in related news- you made me test something and i fixed a bug. :smile: